lib/web/join_tokens_test.go

```diff
@@ -250,11 +255,21 @@ func TestGetTokens(t *testing.T) {
 			resp := GetTokensResponse{}
 			require.NoError(t, json.Unmarshal(re.Bytes(), &resp))
 			require.Len(t, resp.Items, len(tc.expected))
-			require.ElementsMatch(t, resp.Items, tc.expected)
+			require.ElementsMatch(t, filterContentField(resp.Items), filterContentField(tc.expected))
 		})
 	}
 }
 
+func filterContentField(tokens []ui.JoinToken) []ui.JoinToken {
```

Comment:
I added this method to avoid adding a bunch to the tests just to get the yaml content correct